src/reducer.test.js

```diff
@@ -125,4 +149,44 @@ describe('reducer', () => {
       });
     });
   });
+
+  describe('changeLoginField', () => {
+    it('changes changeLoginField', () => {
+      const initialState = {
+        loginFields: {
+          email: '',
+          password: '',
+        },
+      };
+
+      const { name, value } = {
+        name: 'email',
+        value: 'tester@example.com',
+      };
+
+      const state = reducer(initialState, changeLoginField({ name, value }));
```

Comment:
```suggestion
      const { name, value } = {
        name: 'email',
        value: 'tester@example.com',
      };

      const state = reducer(initialState, changeLoginField({ name, value }));
```
destructuring 부분은 코드가 불필요하게 반복되는 것 같아요.
매개변수에 각각 값을 직접 넣어주거나, 차라리 변수를 따로 선언하는게 더 코드 가독성이 좋을 것 같아요.

Reply:
넵~ 감사합니다 수정해보았습니다 👍